web-ui/src/components/menu/Menu.test.jsx

```diff
@@ -84,4 +88,19 @@ describe('<Menu />', () => {
       </AppContextProvider>
     );
   });
+
+  it('adds link to avatar', () => {
+    const { container } = render(
+      <AppContextProvider value={initialState}>
+        <BrowserRouter>
+          <Menu />
+        </BrowserRouter>
+      </AppContextProvider>
+    );
+    const link = container.querySelector('header > a');
+    const href = link.getAttribute('href');
+    const lastIndex = href.lastIndexOf('/');
+    const id = href.substring(lastIndex + 1);
```

Comment:
Another way to grab the `id` could be to split the string into an array and pop off the last element:
```js
const id = href.split('/').pop()
```